src/LibraryViewExtensionWebView2/Handlers/NodeItemDataProvider.cs

```diff
@@ -41,6 +45,7 @@ class NodeItemDataProvider : ResourceProviderBase
     {
         protected NodeSearchModel model;
         private IconResourceProvider iconProvider;
+        readonly string[] typesToHideInHomeWorkspaces = { "Core.Input.Input", "Core.Input.Output" };
```

Comment:
just a fixed list here ? can we use nameof() ? or are the types not accessible in this csproj ?

Reply:
let me take a look, they should be accessible.

Reply:
@pinzart90 PTAL, now using the types directly.